pep_sphinx_extensions/pep_theme/templates/redirect.html

```diff
@@ -0,0 +1,16 @@
+{# Redirect template #}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
```

Comment:
```suggestion
    <meta charset="utf-8">
```
Just a minor nit, but the `/` are unnecessary (also below).

Reply:
I noticed that as well, though it was sufficiently practically unimportant that I didn't mention it (though it did bother me that it was somewhat inconsistently used even on adjacent `void` elements, presumably because some of them were copied from my script while some were copied from the template).

Reply:
> inconsistently used

The `http-eqiv` was the only one missing the closing `/` it seems -- I've now added this. (PS for clarity I didn't copy anything from the script you linked as I thought it could be done in Sphinx internally -- I was reading the MDN documentation and most likely just forgot! Sorry.)

A

Reply:
Why not removing them all then?

Technically it's allowed for HTML5 void elements like `meta` and `link`, but has no effect and it's ignored.
See https://www.w3.org/TR/2011/WD-html-markup-20110113/syntax.html#syntax-elements

Reply:
My HTML auto-formatter is XML based and expects self-closing element. Maybe a bad reason but it is the reason!

A